src/process.ts

```diff
@@ -40,7 +41,8 @@ export class PowerShellProcess {
                     const startScriptPath =
                         path.resolve(
                             __dirname,
-                            "../../scripts/Start-EditorServices.ps1");
+                            this.bundledModulesPath,
+                            "../scripts/Start-EditorServices.ps1");
```

Comment:
So I thought we were just going to leave this to what it was before and then have the build script copy the Start-EditorServices.ps1 and move it under `../../..scripts/Start-EditorServices.ps1`

Reply:
this change moves it under:

`modules/scripts/Start-EditorServices.ps1`

Reply:
Debugger shows the runtime value set here to be `"C:\Users\roholt\Documents\Dev\PowerShellEditorServices\scripts\Start-EditorServices.ps1"` on my machine.

I've got it running against [PSES #647](https://github.com/PowerShell/PowerShellEditorServices/pull/647).

But could be a fluke. Wanna check it works for you too?